.travis.yml

```diff
@@ -1,10 +1,6 @@
-language: node_js
-node_js:
-  - iojs
-
-node_js:
-  - "0.10"
-
+language: erlang
+otp_release:
+   - 17.4
```

Comment:
one potential problem with this is that we can no longer specify the version of node we want to use in the build (afaik). Is there another way to do this? How does the pouchdb project manage to do it?